stdlib/database/mysql/src/main/ballerina/mysql/Module.md

```diff
@@ -4,11 +4,95 @@ This module provides the functionality required to access and manipulate data st
 
 ### Endpoint 
 
-To access a database, you must first create a `client` object. Create a client of the MySQL Client type (i.e., `mysql:Client`) and provide the necessary connection parameters. This will create a pool of connections to the given MySQL database. A sample for creating a MySQL client can be found below.
+To access a database, you must first create a `client` object. A sample for creating a MySQL client can be
```

Comment:
```suggestion
You need to create a `client` object to access a database. The following is a sample for creating a MySQL client.
```

Reply:
I don't think we should say "Following is a ...." because the example is not found immediately.